src/renderer/wg_engine/tvgWgCommon.cpp

```diff
@@ -52,12 +52,12 @@ void WgContext::initialize(WGPUInstance instance, WGPUSurface surface)
     wgpuInstanceRequestAdapter(instance, &requestAdapterOptions, onAdapterRequestEnded, &adapter);
     assert(adapter);
 
-    // adapter enumerate features
+    // adapter enumarate fueatures
```

Comment:
something strange happened since there is many typos...
enumarate, fueatures, reguest